visualizer/parseTree.js

```diff
@@ -436,11 +436,15 @@
       wrapper.classList.add('zoomBorder');
     }
 
-    var label = wrapper.appendChild(createTraceLabel(traceNode));
+    var selfWrapper = wrapper.appendChild(createElement('.self'));
+    var label = selfWrapper.appendChild(createTraceLabel(traceNode));
 
     label.addEventListener('click', function(e) {
       if (e.altKey && !(e.shiftKey || e.metaKey)) {
         console.log(traceNode);  // eslint-disable-line no-console
+      } else if (e.metaKey && !e.shiftKey) {
```

Comment:
Shouldn't we be checking that the alt key is not pressed here? E.g., `e.metaKey && !(e.altKey || e.shiftKey)`